### PR TITLE
add upgrade unit test

### PR DESCRIPTION
**Pull request by hijackthe2**
## Details
add function test_failover_policy to test FailoverPolicy::try_from

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: service/src/upgrade.rs [hunk 1]
```diff
@@ -86,3 +86,41 @@ pub mod fusedev_upgrade {
...
+            FailoverPolicy::try_from("resend").unwrap(),
+            FailoverPolicy::Resend
+        );
+
+        let strs = vec!["flash", "Resend"];
+        for s in strs.clone().into_iter() {
+            assert!(FailoverPolicy::try_from(s).is_err());
+        }
+
+        let str = String::from("flush");
```
Comment (imeoer) [inline service/src/upgrade.rs:110]: Thanks for the test! Is it necessary to test String?

**Comment by imeoer** [comment]
Also please remove the unnecessary commits.

File: service/src/upgrade.rs (hunk 1 above)
Comment (hijackthe2) [reply service/src/upgrade.rs:110]: Thank you for response and suggestion! I test String type transform here to increase code coverage. Indeed, testing String type is optional.

**Comment by imeoer** [comment]
@hijackthe2 Let's remove the unnecessary commits and merge it, thanks!

**Review by imeoer — APPROVED** [review]

**Review by imeoer — APPROVED** [review]

**Pull request merged**